### PR TITLE
add sigma advertising network

### PR DESCRIPTION
**Pull request by foadabdollahi**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9230,12 +9230,22 @@
 		"Adnegah": {
 			"cats": [
 				36
-			],
-			"env": "^ados(?:Results)?$",
```
Comment (jvoisin) [inline src/apps.json]: Why are you removing this?

File: src/apps.json [hunk 2]
```diff
@@ -9230,12 +9230,22 @@
...
+			],"headers": {
+				"X-Advertising-By": "adnegah.net"
```
Comment (jvoisin) [inline src/apps.json]: Would it be possible to add `^` and `$` to the regexp?

File: src/apps.json [hunk 3]
```diff
@@ -9230,12 +9230,22 @@
...
 			"html": "<iframe [^>]*src=\"[^\"]+adnegah\\.net",
 			"icon": "adnegah.png",
 			"script": "[^a-z]adnegah.*\\.js",
 			"website": "Adnegah.net"
+		},
+		"Sigma Advertising": {
+			"cats": [
+				36
+			],"headers": {
+				"X-Advertising-By": "sigmaad.com"
```
Comment (jvoisin) [inline src/apps.json]: Would it be possible to add `^` and `$` to the regexp?

File: src/apps.json [hunk 4]
```diff
@@ -9230,12 +9230,22 @@
...
+			},
+			"icon": "sigmaad.png", 
+			"website": "sigmaad.com"
```
Comment (jvoisin) [inline src/apps.json]: Please specify `https` for the url.

**Comment by AliasIO** [comment]
The indentation was changed from spaces to tabs. Would you mind re-submitting this pull request?

**Comment by AliasIO** [comment]
Closing because of inactivity.

**Pull request closed without merging**